manifests/pulp.pp

```diff
@@ -129,50 +134,79 @@
     }
   }
 
+  if $expose_mongodb {
+    include certs::mongodb
+
+    Class['certs::mongodb'] -> Class['mongodb::server']
```

Comment:
You can also add a `require => Class['certs::mongodb']` on the `class { 'mongodb::server':` definition`.